src/main/java/org/commoncrawl/stormcrawler/news/NewsSiteMapParserBolt.java

```diff
@@ -32,6 +32,7 @@
 import org.apache.storm.task.TopologyContext;
 import org.apache.storm.tuple.Tuple;
 import org.apache.storm.tuple.Values;
+import org.commoncrawl.stormcrawler.news.bootstrap.FeedDetectorBolt;
```

Comment:
Maybe it would be better to move FeedDetectorBolt into the package org.commoncrawl.stormcrawler.news, now as it is not only used in the bootstrap topology.